### ISSUE TITLE
Toasts and doAsync{}, uiThread, function calls no longer work with Android studio 3.3

### ISSUE THREAD

**Issue by ozzy4654** [Kotlin/anko#666]
gradle-4.9-all.zip
classpath 'com.android.tools.build:gradle:3.3.0-alpha06'

doAsync{}/uiThread{}
Cannot access class 'Ljava.util.concurrent.Future;'. Check your module classpath for missing or conflicting dependencies
Cannot access class 'Lkotlin.Function1;'. Check your module classpath for missing or conflicting dependencies
Type inference failed: fun <T> T.doAsync(exceptionHandler: `Function1;`<Throwable, Unit>? = ..., task: `Function1;`<`AnkoAsyncContext;`<T>, Unit>): `Future;`<Unit>

Toasts:

Unresolved reference. None of the following candidates is applicable because of receiver type mismatch: 
public inline fun `Fragment;`.toast(message: CharSequence): `Toast;` defined in org.jetbrains.anko
public inline fun `Fragment;`.toast(message: Int): `Toast;` defined in org.jetbrains.anko
public inline fun `Context;`.toast(message: CharSequence): `Toast;` defined in org.jetbrains.anko
public inline fun `Context;`.toast(message: Int): `Toast;` defined in org.jetbrains.anko
public inline fun `AnkoContext;`<*>.toast(message: CharSequence): `Toast;` defined in org.jetbrains.anko
public inline fun `AnkoContext;`<*>.toast(message: Int): `Toast;` defined in org.jetbrains.anko
> Task :commons-android:compileDebugKotlin FAILED

**Comment by erniealice** [comment]
If you're comfortable with Canary builds, you can still use Android Studio 3.3 Canary Build 5. The errors occur when using the Canary Build 6 and Gradle Alpha 6.

Also experiencing the same problem. IDE was throwing some errors though I haven't really changed my code, except for the  necesaary`build.gradle` to match Gradle dependencies (Alpha 6).

To add, `Context.alert` functions to do not seem to work as well.

**Comment by OleksiiAndroid** [comment]
Same error...

**Comment by krishna-incrypto** [comment]
Yeah Iam at Android Studio 3.4 and still they havent found a sloution to this issue its driving me crazy.Still doAsync and uiThread dont work.
